vignettes/vignettes.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "StructuralVariantAnnotation Quick Overview"
+author: "Ruining Dong"
+date: "`r Sys.Date()`"
+output:
+  pdf_document:
+    toc: yes
+  html_document:
+    toc: yes
+    toc_float:
+      collapsed: yes
+      smooth_scroll: yes
+vignette: |
+  %\VignetteIndexEntry{Vignette Title} %\VignetteEngine{knitr::rmarkdown} %\VignetteEncoding{UTF-8}
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(#echo = TRUE,
+  collapse = TRUE,
+  comment = "#>")
+```
+
+## Introduction
+This vignette outlines a workflow of parsing and plotting structural variants from Variant Call Format (VCF) using the `StructuralVariantAnnotation` package. `StructuralVariantAnnotation` contains useful helper functions for reading and interpreting structural variants calls. The packages contains functions for parsing VCFs from a number of popular caller as well as functions for dealing with breakpoints involving two separate genomic loci encoded as `GRanges` objects. 
+
+## Installation
+
+The *StructuralVariationAnnotation* package can be loaded from *Bioconductor* as follows:
+
+```{r installation, eval=FALSE}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("StructuralVariantAnnotation")
+```
+
+## Breakpoint notation revisited
```

Comment:
The package supports all 3 VCF notations. We could merge this with the GRanges explanation section and put the examples of the various VCF notations, and how the all nicely encode to the same GRanges representation here.